src/RequestInformation.cs

```diff
@@ -203,13 +203,15 @@ private static object ReplaceEnumValueByStringRepresentation(object source)
         {
             var type = value.GetType();
 
-            if(Enum.GetName(type, value) is not { } name)
+            if (Enum.GetName(type, value) is not { } name)
                 throw new ArgumentException($"Invalid Enum value {value} for enum of type {type}");
 
-            if(type.GetField(name)?.GetCustomAttribute<EnumMemberAttribute>() is { } attribute)
-                return attribute.Value;
+            [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2075:UnrecognizedReflectionPattern",
+                Justification = "Enumerating Enum fields is always trimming/AOT safe - https://github.com/dotnet/runtime/issues/97737")]
```

Comment:
Looks like my original suggestion didn't build. Try this:

```suggestion
#if NET5_0_OR_GREATER
            [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2070:UnrecognizedReflectionPattern",
                Justification = "Enumerating Enum fields is always trimming/AOT safe - https://github.com/dotnet/runtime/issues/97737")]
#endif

```

Reply:
Good catch, just for my reference, is the change from 2075 to 2070 just based on where the type object is originating. (method return vs parameter)?

Reply:
Correct. See

https://github.com/dotnet/runtime/blob/main/docs/tools/illink/error-codes.md#il2070-trim-analysis-this-argument-does-not-satisfy-dynamicallyaccessedmembersattribute-in-call-to-target-method-the-parameter-source-parameter-of-method-source-method-does-not-have-matching-annotations-the-source-value-must-declare-at-least-the-same-requirements-as-those-declared-on-the-target-location-it-is-assigned-to

vs

https://github.com/dotnet/runtime/blob/main/docs/tools/illink/error-codes.md#il2075-trim-analysis-this-argument-does-not-satisfy-dynamicallyaccessedmembersattribute-in-call-to-target-method-the-return-value-of-method-source-method-does-not-have-matching-annotations-the-source-value-must-declare-at-least-the-same-requirements-as-those-declared-on-the-target-location-it-is-assigned-to